styles.css

```diff
@@ -615,7 +614,164 @@
   display: none;
 }
 
+/* popup */
+.popup-details {
+  position: fixed;
```

Comment:
Thanks for your review.
![Screenshot (30)](https://user-images.githubusercontent.com/117263981/216147646-c72d0530-1bce-40d7-98a1-259817f8d9e4.png)

kindly suggest what should I do with it. because these problems are not here on my side.
![Screenshot (29)](https://user-images.githubusercontent.com/117263981/216147627-09d51cc9-9bc4-4555-8ec5-11fa555cbbe9.png)

Reply:
@BregornOriginal kindly I am waiting for your response.

Reply:
@BregornOriginal Hi, Thank you for your review but we are sticking here because this part of the website looks good in my browser. we don't know what we should do because we don't know where the problem . 
thanks